include/libdnf5/base/log_event.hpp

```diff
@@ -41,30 +41,32 @@ class LogEvent {
         libdnf5::GoalProblem problem,
         const std::set<std::string> & additional_data,
         const libdnf5::GoalJobSettings & settings,
-        const libdnf5::transaction::TransactionItemType spec_type,
+        libdnf5::transaction::TransactionItemType spec_type,
```

Comment:
Why there is a different signature than in cpp? Meaning the `const` is missing here. Is it on purpose?

Reply:
I also got this from clang-tidy: [readability-avoid-const-params-in-decls](https://clang.llvm.org/extra/clang-tidy/checks/readability/avoid-const-params-in-decls.html)

The way I understand it is that since the `spec_type` is passed by value marking it `const` in the declaration has no effect, the passed value is only copied.
On the other hand in the definition it ensures that the copied valued is not modified.

I would be fine leaving both cases unmodified, reverting my changes, if you prefer.

Reply:
I don't have a problem with either way, just I'd be for having the hpp and cpp in sync.

Reply:
Ok, I have added it back to the hpp.